### ISSUE TITLE
Embed SonarCSharp 8.2

### ISSUE THREAD

**Issue by andrei-epure-sonarsource** [SonarSource/sonarlint-visualstudio#1191]
Release notes: https://github.com/SonarSource/sonar-dotnet/releases/tag/8.2.0.14119

(note: there's an FP fix in addition to the CBDE integration)

**Comment by duncanp-sonar** [comment]
Note: v8.2 has the following additional dependencies:

### Assembly: SonarAnalyzer.CSharp.dll
```
System.Runtime.InteropServices.RuntimeInformation, Version=4.0.1.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a
System.Xml, Version=4.0.0.0, Culture=neutral, PublicKeyToken=
System.Xml.Linq, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089
```

These look pretty standard so it should be ok, but specific testing on VS2015.3 will be required.

It seems weird that `System.Runtime.InteropServices.RuntimeInformation` is now required. Still, as long as it's available it doesn't really matter.

**Comment by duncanp-sonar** [comment]
Manually tested on vanilla VS2015.3, vanilla VS2017 and latest VS2019.

**Comment by duncanp-sonar** [comment]
Blocked by https://github.com/SonarSource/sonar-dotnet/issues/3048

**Comment by duncanp-sonar** [comment]
Marking this as won't fix. We'll embed a version that has a fix for [sonar-dotnet-3048](https://github.com/SonarSource/sonar-dotnet/issues/3048) which will have a different version number.
